constant.go

```diff
@@ -0,0 +1,29 @@
+package gini18n
```

Comment:
change package name to `i18n`

Reply:
@appleboy I updated code from your comment. Please review these, thanks!